CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Update canonical to `v0.6` [#72](https://github.com/dusk-network/phoenix-core/issues/72)
 - Crossover: use all attributes as hash inputs [#69](https://github.com/dusk-network/phoenix-core/issues/69)
 - Message: use all attributes as hash inputs [#69](https://github.com/dusk-network/phoenix-core/issues/69)
```

Comment:
When those were added? I know this is not part of this PR, but the format is not consistent with the changelog and therefore it should be fixed. We can do it before the release, at this point. But keep that in mind for the future.

Reply:
What is the exact issue with the format?

Reply:
That should start wit the `Update`, `Add`, `Remove` etc.. words.